tests/db_engine_specs/mysql_tests.py

```diff
@@ -70,7 +70,7 @@ def test_is_db_column_type_match(self):
             ("TINYINT", GenericDataType.NUMERIC),
             ("SMALLINT", GenericDataType.NUMERIC),
             ("MEDIUMINT", GenericDataType.NUMERIC),
-            ("INT", GenericDataType.NUMERIC),
+            ("INTEGER", GenericDataType.NUMERIC),
```

Comment:
Isn't `INT` an official datatype in Mysql? https://dev.mysql.com/doc/refman/8.0/en/integer-types.html I think we need to keep this here.